package.json

```diff
@@ -10,7 +10,7 @@
     "pontoon-to-json": "index.js"
   },
   "scripts": {
-    "test": "eslint --config ./.eslintrc.yaml '*.js'"
+    "test": "eslint ."
```

Comment:
Simplified this to just `eslint .` since it will load an [.eslintrc.yaml file by default](http://eslint.org/docs/user-guide/configuring#configuration-file-formats).